packages/entity-list/src/util/api/forms.js

```diff
@@ -37,6 +40,16 @@ export const columnDefinitionTransformer = json => {
       child: c.children.filter(isDisplayableChild)[0]
     }
   ))
+
+  if (table.sorting) {
+    const sorting = table.sorting.reduce((acc, val, idx, arr) => {
```

Comment:
Have you considered using map instead of reduce? its easier to read imo

const sorting = table.sorting.map(sort => `${sort.orderItem} ${(sort.ascending) ? 'asc' : 'desc'}`).join(',')